ServiceProvider/RecommendServiceProvider.php

```diff
@@ -131,7 +131,7 @@ public function register(BaseApplication $app)
         }));
 
         // initialize logger (for 3.0.0 - 3.0.8)
-        if (version_compare(Constant::VERSION, '3.0.8', '<=')) {
+        if (Version::isSupport('3.0.8', '<=')) {
```

Comment:
use `!Version::isSupportGetInstanceFunction()`
(notice: eccube_log_init runs, if not support GetInstanceFunction)